activesupport/lib/active_support/testing/strict_warnings.rb

```diff
@@ -10,6 +10,7 @@ class WarningError < StandardError; end
     PROJECT_ROOT = File.expand_path("../../../../", __dir__)
     ALLOWED_WARNINGS = Regexp.union(
       /circular require considered harmful.*delayed_job/, # Bug in delayed job.
+      /circular require considered harmful.*rails-html-sanitizer/, # Bug when sprockets-rails is not required.
```

Comment:
If there is a bug in one of Rails' gems, why aren't fixing instead of delaying the problem? Who is expected to fix this?